xonsh/proc.py

```diff
@@ -2033,7 +2033,8 @@ def iterraw(self):
         # read from process now that it is over
         yield from safe_readlines(stdout)
         self.stream_stderr(safe_readlines(stderr))
-        proc.wait()
+        if not ON_CYGWIN:
+            proc.wait()
```

Comment:
This doesn't seem correct. This seems like it would have processes return before they are finished running.

Reply:
Well, I wasn't sure about it, but that was where simple shell commands were hanging.  The command (e.g. `ls`) would output as expected, then pause forever in that `proc.wait()`.  I'm not saying it it correct, or that I understand it, but cygwin liked it that way.  Is there some other flag/semaphore that would be a better choice?

Reply:
Can you try these commands with [`$THREAD_SUBPROCS=False`](https://xon.sh/envvars.html#thread-subprocs), and see if wait() still hangs?

Reply:
That works fine!  With that set in the rc file the `proc.wait()` can be left as it was.  Thank you.

Reply:
Yeah, we may want to change the default for cygwin to False as a better fix than this

Reply:
Yes, that seems like a more appropriate fix; I just don't see where the default is set.

Reply:
It is in environ.py

Reply:
Ok, corrected in environ.py.  Btw, it passes all checks but that pesky "lint/xonsh" one, which doesn't indicate why the test fails.  How can I correct that?

Reply:
Hi @wendellwt, that pesky test says literally "No news item" and, as @scopatz already pointed out, there's indeed no news entry. Could you please add one? See [Development Guide](https://xon.sh/devguide.html#changelog) for details.

Reply:
Yeah you need to add a news item, see https://xon.sh/devguide.html#changelog

Reply:
Oh, _that_ news item.  Done.  Sorry about that.